juju/juju
# fix: checking for hooks in local charm

When checking if a local charm has hooks, we were assuming that the zip file would always have a separate entry for the directory `hooks/` and for each of its subfiles. However, this is not the case for charms generated by Charmcraft. A better check is that there exists at least one entry with the prefix `hooks/` that is *not* a directory.

## Checklist




## QA steps

Check existing unit tests continue to pass.

Pack the `departer` charm (`testcharms/charms/departer`) and verify that it can be deployed.
